l10n_ch_payment_slip/report/report.py

```diff
@@ -81,6 +81,7 @@ def _generate_one_slip_per_page_from_invoice_pdf(self, report_name=None,
                             doc.invoice_id.id),
                         'res_model': save_in_attachment.get('model'),
                         'res_id': doc.invoice_id.id,
+                        'type':'binary'
```

Comment:
```suggestion
                        'type': 'binary'
```

Just a missing space in fact.